monitor/ui/src/main/webapp/scheduling.xhtml

```diff
@@ -57,7 +55,7 @@
 						<span class="glyphicon glyphicon-plus" aria-hidden="true" /> <span>Build
 							job</span>
 					</button>
-					<form action="repository/#{user.tenant}/jobs/#{file.name}"
+					<form action="repository/#{user.tenant}/jobs/multiple_upload"
```

Comment:
What does "#{file.name}" refer to? The value was (at least in my case) an empty string and I did not find any "File"Bean.java. Does it have any purpose or is/was that just some leftover?